hack/crds-sync.sh

```diff
@@ -17,3 +18,18 @@ done
 
 #this one goes in manifests rather than install, but should it? 
 cp "vendor/github.com/openshift/api/machineconfiguration/v1/0000_80_controllerconfig.crd.yaml" "manifests/controllerconfig.crd.yaml"
+cp "vendor/github.com/openshift/api/machineconfiguration/v1alpha1/0000_80_machineconfignode-TechPreviewNoUpgrade.crd.yaml" "manifests/0000_80_machine-config-operator_01_machineconfignode-TechPreviewNoUpgrade.crd.yaml" 
+cp "vendor/github.com/openshift/api/operator/v1/0000_80_machine-config-operator_01_config.crd.yaml" "install/0000_80_machine-config-operator_01_config.crd.yaml"
+
+
+#v1/0000_10_containerruntimeconfig.crd.yaml:0000_80_machine-config-operator_01_containerruntimeconfig.crd.yaml
+#v1/0000_10_kubeletconfig.crd.yaml:0000_80_machine-config-operator_01_kubeletconfig.crd.yaml
+#v1/0000_10_machineconfig.crd.yaml:0000_80_machine-config-operator_01_machineconfig.crd.yaml
+#v1/0000_10_machineconfigpool.crd.yaml:0000_80_machine-config-operator_01_machineconfigpool.crd.yaml
+#TODO(jkyros): 0000_80_machine-config-operator_02_containerruntimeconfig.crd.yaml
+
+#./vendor/github.com/openshift/api/machineconfiguration/
+#./vendor/github.com/openshift/api/machineconfiguration/
+#./vendor/github.com/openshift/api/machineconfiguration/
+#./vendor/github.com/openshift/api/machineconfiguration/
+#./vendor/github.com/openshift/api/machineconfiguration/v1/
```

Comment:
Hmmm this comment looks familiar, but it never made it into master as part of https://github.com/openshift/machine-config-operator/pull/3747. Does this need a rebase, or did something else happen? :smile: 

(Relevant conversation from API merge: https://github.com/openshift/machine-config-operator/pull/3747#discussion_r1350320561)

Reply:
my changes might have inadvertently overwritten yours. Let me look at this...

Reply:
yep, I will remove this